### PR TITLE
Made changes to order add logic for PS 1770, by Lukas

### PR DESCRIPTION
**Pull request by tomjas1997**
Just taking some changes from https://github.com/Invertus-mollie/PrestaShop/pull/164/files

### CODE REVIEW

File: mollie.php [hunk 1]
```diff
@@ -530,9 +530,11 @@ public function hookActionAdminControllerSetMedia()
             $this->context->controller->addCSS($this->getPathUri() . 'views/css/admin/order-list.css');
             $this->context->controller->addJS($this->getPathUri() . 'views/js/admin/order_list.js');
 
-            if (Tools::isSubmit('addorder')) {
+            //TODO: consider removed this if statement
```
Comment (tomas862) [inline mollie.php:533]: why not removing it now ?

File: mollie.php [hunk 2]
```diff
@@ -563,7 +565,9 @@ public function hookDisplayBackOfficeHeader()
                 'mollieCheckMethods' => time() > ((int)Configuration::get(Mollie\Config\Config::MOLLIE_METHODS_LAST_CHECK) + Mollie\Config\Config::MOLLIE_METHODS_CHECK_INTERVAL),
             ]);
             $html .= $this->display(__FILE__, 'views/templates/admin/ordergrid.tpl');
-            if (Tools::isSubmit('addorder')) {
+
+            //TODO: consider removed this if statement
```
Comment (tomas862) [inline mollie.php:569]: same here

File: views/js/admin/order_add.js [hunk 3]
```diff
@@ -32,32 +32,34 @@
...
         var selectedPayment = $(this).val();
         var isMollie = isMolliePayment(selectedPayment);
-
         toggleOrderStatus(isMollie);
     });
 
     function toggleOrderStatus(isMolliePayment) {
         var $molliePaymentCheckboxGroup = $('#mollie-email-send-group');
-        var $orderStatusSelector = $('select[name="id_order_state"]');
+        var $orderStatusSelector = isPsVersion177 ? $('select[name="cart_summary[order_state]"]') : $('select[name="id_order_state"]');
```
Comment (tomas862) [inline views/js/admin/order_add.js:53]: I think you can avoid conditions just by comma separating multiple values in jQuery.

Like this:

```javascript
$('select[name="cart_summary[order_state]"],select[name="id_order_state"]')
```
does this work ?

Comment (tomjas1997) [reply views/js/admin/order_add.js]: Seems to work, changed it.

**Review by tomas862 — APPROVED** [review]

**Pull request merged**